server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -54,19 +55,28 @@ object CORS {
         else
           config.exposedHeaders.map(headerFromStrings("Access-Control-Expose-Headers", _))
 
+      def checkVaryHeader(response: Response[G]): Option[Header] = {
+        val maybeHeader = response.headers.get(CaseInsensitiveString("Vary"))
+        if (maybeHeader.isEmpty) Option(Header("Vary", "Origin,Access-Control-Request-Method"))
```

Comment:
Microoptimization: this default header could be stored in a val and shared by all requests.

Reply:
A microoptimization suggested by Ross himself?!

Are my repeated pedantic comments on microoptimizations finally making a dent?